packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -542,6 +542,13 @@ export class ModelScene extends Scene {
 
     if (name != null) {
       animationClip = this.animationsByName.get(name);
+      
+      if (animationClip == null) {
+        const parsedAnimationIndex = parseInt(name);
+
+        if (!isNaN(parsedAnimationIndex) && parsedAnimationIndex >= 0 && parsedAnimationIndex < animations.length) {
+          animationClip = animations[parsedAnimationIndex];
+        }
```

Comment:
I think you missed a `}`

Reply:
🤦‍♂️ the wonders of incomplete dev environments and partial discards. Thanks!

Got a related question regarding dev env: it seems that `npm run dev` in packages/model-viewer now always runs karma; I think it used to serve modelviewer.dev for testing instead. What's the right steps nowadays to work on model-viewer but test in modelviewer.dev?

Reply:
So, someone recently changed the `dev` script, but I didn't pay a lot of attention because I never use it personally. I just have one terminal open where I have `npm run serve` going non-stop, and then another terminal where I do `npm run build:dev` and `npm test` alternately in the model-viewer package directory. If you'd like the script to behave differently (or add others) feel free to toss up a PR. I'm not sure what other devs like in terms of automation.